Dockerfile

```
@@ -0,0 +1,33 @@
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    g++ \
+    make \
+    wget \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install CmdStan
+WORKDIR /opt
+RUN wget https://github.com/stan-dev/cmdstan/releases/download/v2.33.1/cmdstan-2.33.1.tar.gz \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)